install_config/configuring_sdn.adoc

```diff
@@ -183,6 +192,27 @@ networkConfig:
 *redhat/openshift-ovs-networkpolicy* for the *ovs-networkpolicy* plug-in
 ====
 
+[[growing-the-service-network]]
+== Growing the Service Network
+
+If you are running low on addresses in your Service network and can expand the range so that it includes the current range at the beginning of the new range, then you can:
+
+. Change the `*serviceNetworkCIDR*` parameter on all
+xref:configuring-the-pod-network-on-masters[masters] in their configuration files.
+ifdef::openshift-origin[].  You can only change the number following the / to a smaller number.
```

Comment:
Is the statement after the ifdef on this line for Origin? Or is the ifdef for the stuff below? Sounds like it could be with the "step 1" above.